### PR TITLE
test(amplify-migration-tests): fix cci test failures, add version helper

### PR DESCRIPTION
**Pull request by jhockett**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-migration-tests/src/migration-helpers/check-version.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { getCLIPath, nspawn as spawn } from 'amplify-e2e-core';
+
+export function versionCheck(cwd: string, testingWithLatestCodebase = false) {
```
Comment (SwaySway) [inline packages/amplify-migration-tests/src/migration-helpers/check-version.ts:3]: nit: Leftover Function?

**Review by SwaySway — APPROVED** [review]
LGTM aside from nit

File: packages/amplify-migration-tests/src/migration-helpers/check-version.ts (hunk 1 above)
Comment (SwaySway) [reply packages/amplify-migration-tests/src/migration-helpers/check-version.ts:3]: synced offline

Comment (jhockett) [reply packages/amplify-migration-tests/src/migration-helpers/check-version.ts:3]: intentional for future use

**Pull request merged**